spec/models/order_spec.rb

```diff
@@ -5,4 +5,8 @@
   it { should belong_to :user }
   it { should have_many :line_items }
   it { should have_many(:items).through(:line_items) }
+  it do
+    should validate_inclusion_of(:status).
```

Comment:
@erinnachen @brennanholtzclaw I'm having issues getting rid of this hound error and also with rebasing all of these hound fixes. This branch should be good to merge into master otherwise. can one of you please take a look at rebasing the hound commits and merge when ready? thanks!